app/services/idv/session.rb

```diff
@@ -82,16 +82,16 @@ def phone_confirmed?
     end
 
     def complete_session
+      associate_in_person_enrollment_with_profile
+
       if address_verification_mechanism == 'gpo'
         profile.deactivate(:gpo_verification_pending)
         create_gpo_entry
       elsif phone_confirmed?
         if pending_in_person_enrollment?
-          UspsInPersonProofing::EnrollmentHelper.new.save_in_person_enrollment(
+          UspsInPersonProofing::EnrollmentHelper.schedule_in_person_enrollment(
             current_user,
-            profile,
             applicant,
```

Comment:
Would you be alright if we followed-up on this separately. I agree with your worry, and I've observed elsewhere we're not very consistent with passing the struct vs. hash, so maybe we could normalize this to a single argument type (e.g. pass `pii` here instead of `applicant`).

Reply:
I created LG-7123 for this